docs/Users.mdx

```diff
@@ -0,0 +1,10 @@
+---
+id: users
+title: Users
+---
+
+These are the user guides to consuming the typescript-eslint tooling.
+It's intended for users who want to consume our tools.
+
+> If you're reading this as a new user: welcome to the community!
+> We're happy to have you! ❤️‍🔥
```

Comment:
I suggest not using block quotes for callouts.

```suggestion
:::info Welcome!
If you're reading this as a new user: welcome to the community!
We're happy to have you! ❤️‍🔥
:::
```

Reply:
Copy 🍝
Happy to improve it!